src/platform-includes/enriching-events/user-feedback/sdk-api-example/react-native.mdx

```diff
@@ -2,7 +2,8 @@
 import * as Sentry from '@sentry/react-native';
 import { UserFeedback } from '@sentry/react-native';
 
-const sentryId = Sentry.captureMessage('My Message');
+const sentryId = Sentry.lastEventId();
+// OR: const sentryId = Sentry.captureMessage('My Message');
```

Comment:
```suggestion
const sentryId = Sentry.captureMessage('My Message');
// OR: const sentryId = Sentry.lastEventId();
```

Reply:
`lastEventId` might be null or invalid if there's no event captured before using this function.